app/views/content/terms.md

```diff
@@ -28,39 +28,31 @@ The following definitions are used in these Terms:
 
 1.11 “Documentation” means all documentation and/or instructions that we may make available with the API, including but not limited to any information that we post on the developers section of the Artsy website.
 
-1.12 "Production" means distributing, licensing, or otherwise making Your Application available to any third party, for free or otherwise. 
-
-1.13 “on the Services” means on the API and/or on the Artsy website.
+1.12 “on the Services” means on the API and/or on the Artsy website.
 
 ##2. Basic Terms
 
-2.1 You may use the API only if you can form a binding contract with us, and only as permitted by law. You are not permitted to use the API if you are a person under the age of eighteen (18).
-
-2.2 If you are accepting these Terms and using the API on behalf of a company, organization, or other legal entity, you represent and warrant that you are authorized to bind that entity to these Terms, in which case the terms “you” and “your” in these Terms will refer to that entity. 
-
-2.3 Your use of the API is also subject to our general [Terms of Use](https://artsy.net/terms) and [Privacy Policy](https://artsy.net/privacy) as amended by us from time to time. By accessing or using the Services in any manner, you agree to be bound by these terms and conditions. 
-
-2.4 Subject to these Terms, we grant you a limited, worldwide (except as limited by these Terms), non-exclusive, non-sublicensable, non-transferable, revocable license to use the API in the manner permitted by these Terms. This license will terminate automatically if you breach any of these Terms. We may also terminate this license at any time for any reason, as provided in section 18 of these Terms. 
+2.1 You may use the API only if you can form a binding contract with us, and only as permitted by law. You are not permitted to use the API if you are a person under the age of eighteen (18). If you are accepting these Terms and using the API on behalf of a company, organization, or other legal entity, you represent and warrant that you are authorized to bind that entity to these Terms, in which case the terms “you” and “your” in these Terms will refer to that entity. 
 
-2.5 You agree to register for an Artsy account and provide all required information. You agree to provide only true and accurate information in connection with your account, and to update all such information as necessary to keep it accurate and current. 
+2.2 Your use of the API is also subject to our general [Terms of Use](https://artsy.net/terms) and [Privacy Policy](https://artsy.net/privacy) as amended by us from time to time. By accessing or using the Services in any manner, you agree to be bound by these terms and conditions. 
```

Comment:
Fix this link, add `www.`, so https://www.artsy.net/terms. It redirects, but useful for SEO.